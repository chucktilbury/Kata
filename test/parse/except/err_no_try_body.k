;-------------------------------------------------------------------------------
; try body absent
;-------------------------------------------------------------------------------

func name1()() {

    try 
        name2 = "string"
    except(name3, name4) {
        name5 = 123
    }
    
}
