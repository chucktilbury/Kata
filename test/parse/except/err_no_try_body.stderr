Syntax: /home/chuck/Src/Kata/test/parse/except/err_no_try_body.k: 8: 14: expected a function body but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/except/err_no_try_body.k: 9: 17: expected function body element but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/except/err_no_try_body.k: 10: 14: expected function body but got symbol
